PowerPointLabs/PowerPointLabs/ActionFramework/Action/ColorsLabActionHandler.cs

```diff
@@ -0,0 +1,20 @@
+﻿using PowerPointLabs.ActionFramework.Common.Attribute;
+using PowerPointLabs.ActionFramework.Common.Extension;
+using PowerPointLabs.ActionFramework.Common.Interface;
+
+namespace PowerPointLabs.ActionFramework.Action
+{
+    [ExportActionRibbonId("ColorsLabButton")]
+    class ColorsLabActionHandler : ActionHandler
+    {
+        protected override void ExecuteAction(string ribbonId)
+        {
+            var colorPane = 
+                this.RegisterTaskPane(typeof(ColorPane), TextCollection.ColorsLabTaskPanelTitle);
```

Comment:
type `this.` and then you can access the ppt context provided by Action Framework